tcf_website/templates/browse/browse.html

```diff
@@ -15,7 +15,7 @@
         {% include "../common/leaderboard_ad.html" with ad_slot="9691204298" %}
 
         <!-- Browse page content -->
-        <h2 class="font-weight-light mt-4">Browse by Department</h2>
+        <h2 class="font-weight-light mt-4" id="browse-title">Browse by Department</h2>
```

Comment:
Had to add id's to certain elements or else it was impossible for selenium to access them